build.toml

```diff
@@ -53,22 +47,9 @@ project = '{cd}'
 target = 'RT CompactRIO Target - VxWorks'
 build_spec = 'Engine Release'
 
-[[build.steps]]
-name = 'FPGA IP'
-type = 'lvBuildSpec'
-project = '{ip}'
-target = 'My Computer'
-build_spec = 'FPGA IP Distribution'
-
-[[build.steps]]
-name = 'SysDef API'
-type = 'lvBuildSpec'
-project = '{api}'
-target = 'My Computer'
-build_spec = 'System Definition API'
-
 [package]
 type = 'nipkg'
+payload_dir = 'Built'
 name = 'ni-fpga-addon-{veristand_version}'
-dev_xml_path = 'Source\Addon\Custom Device FPGA Addon.xml'
```

Comment:
Why is this line removed?

Reply:
The build.toml file was a copy from the original repo with incorrect paths and tags. 
I checked in the official toml documentation and the tag dev_xml_path does not appear as a valid tag for building packages.(for more details please see https://github.com/ni/niveristand-custom-device-build-tools/blob/master/docs/Toml%20Help.md)